dev/src-expose/main.go

```diff
@@ -0,0 +1,172 @@
+// Command "src-expose" serves git repositories within some directory over HTTP,
```

Comment:
That's now outdated, I think

```suggestion
// Command "src-expose" serves directories as git repositories over HTTP,
```

Or something like that :)